src/functions/styling.js

```diff
@@ -49,15 +49,6 @@ defineFunction({
     },
     mathmlBuilder(group, options) {
         // Figure out what style we're changing to.
-        // TODO(kevinb): dedupe this with buildHTML.js
-        // This will be easier of handling of styling nodes is in the same file.
-        const styleMap = {
-            "display": Style.DISPLAY,
-            "text": Style.TEXT,
```

Comment:
I didn't realize this was defined earlier in this file.  Thanks for cleaning this up.